src/Nethermind/Nethermind.TxPool/Filters/BalanceTooLowFilter.cs

```diff
@@ -62,9 +69,17 @@ public AcceptTxResult Accept(Transaction tx, TxFilteringState state, TxHandlingO
 
             if (balance < cumulativeCost)
             {
+                Metrics.PendingTransactionsTooLowBalance++;
+
                 if (_logger.IsTrace)
+                {
                     _logger.Trace($"Skipped adding transaction {tx.ToString("  ")}, insufficient funds.");
-                return AcceptTxResult.InsufficientFunds.WithMessage($"Account balance: {balance}, cumulative cost: {cumulativeCost}");
+                    return AcceptTxResult.InsufficientFunds.WithMessage($"Account balance: {balance}, cumulative cost: {cumulativeCost}");
```

Comment:
Those messages are useful for local transactions as they will be returned to the user. We can allocate them only for local ones though. This applies to all filters.

Reply:
Changed to output the good messages for local ones